config/server.js

```diff
@@ -10,7 +10,7 @@ const config = {
     pgUri: process.env.DATABASE_URL || 'postgres://localhost:5432/concaria'
   },
   habanero: {
-    host: process.env.HABANERO_HOST
+    embedded: process.env.HABANERO_EMBEDDED || true
```

Comment:
the always true is because we are going to default to ON.  If someone wants to disable the orchestrator/habanero they can set it to `false`